### ISSUE TITLE
Support MongoDB Connection Options

### ISSUE THREAD

**Issue by Ahmadre** [prisma/prisma#20751]
## Problem

I can't pass several options I need to setup for my `MongoClient` for a secure connection:

https://www.mongodb.com/docs/manual/core/csfle/fundamentals/automatic-encryption/#enabling-automatic-client-side-field-level-encryption

## Suggested solution

it should be possible to pass to your db connector also the mongo options (see link above).

**Comment by janpio** [comment]
Is this request only about (what in the Node.js MongoDB Client would be) `autoEncryption`? Which options _concretely_ do you require?

**Comment by Ahmadre** [comment]
> Is this request only about (what in the Node.js MongoDB Client would be) `autoEncryption`? 
> 
> Which options _concretely_ do you require?

Exactly. The same like mongoose can connect as a secure client with autoEncryption, it would be great if prisma could also do that.

**Comment by MikePodgorniy** [comment]
Please add compressors options – https://github.com/prisma/prisma/discussions/25750

**Comment by MikePodgorniy** [comment]
**mongo-rust-driver** https://github.com/prisma/mongo-rust-driver based on official mongodb-rust-driver with full params support, just unlock it or say how to do it, please
